src/site/stages/build/drupal/graphql-tome-differences/block_content.md

```diff
@@ -0,0 +1,118 @@
+# block_content
+
+Tome returned **28** records.
+
+
+## Missing properties in Tome:
+
+1. `"fieldLinkSummary": "Download letters like your eligibility or award letter for certain benefits."`
+1. `"url": { "path": "/records/download-va-letters" },`
+
+## Example GraphQL response
+
+```json
+{
+  "entity": {
+    "fieldLink": {
+      "url": {
+        "path": "/records/download-va-letters"
+      },
+      "title": "Confirm your VA benefit status",
+      "options": []
+    },
+    "fieldLinkSummary": "Download letters like your eligibility or award letter for certain benefits."
```

Comment:
This appears to be called `field_promo_link` which is a reference to paragraph.69620159-bb03-45bc-915e-35147050d1d8.json and is just declared differently in the GraphQL query.